srcpkgs/sequoia-pgp/template

```diff
@@ -0,0 +1,21 @@
+pkgname=sequoia-pgp
+version=0.16.0
+revision=1
+short_desc="A cool OpenPGP library and tool written in Rust"
+homepage="https://sequoia-pgp.org/"
+license=('GPL-2.0-or-later')
+maintainer="notramo <notramo@protonmail.com>"
+build_style=gnu-makefile
+makedepends="make pkg-config clang gcc rust cargo git nettle-devel libressl-devel sqlite-devel capnproto python3 python3-devel python3-setuptools python3-pytest python3-cffi"
+distfiles=("https://gitlab.com/sequoia-pgp/sequoia/-/archive/v${version}/sequoia-v${version}.tar.gz")
+checksum=0e9b8f5efda5a1f74b1d70c52a362b95267b7fa3de4d96563605c717bf2a9b57
+wrksrc="${pkgname}-v${version}"
```

Comment:
this is default, don't set it

Reply:
this looks ok, notice the v, which is not the default.

Reply:
indeed